libraries/AP_Arming/AP_Arming.h

```diff
@@ -38,6 +38,7 @@ class AP_Arming {
         ARMING_CHECK_AUX_AUTH    = (1U << 17),
         ARMING_CHECK_VISION      = (1U << 18),
         ARMING_CHECK_FFT         = (1U << 19),
+        ARMING_CHECK_OSD         = (1U << 11),
```

Comment:
```suggestion
        ARMING_CHECK_OSD         = (1U << 20),
```

remember we only have ~28 of these, however :-)

Reply:
sorry for the typo